src/bluesky/plans.py

```diff
@@ -21,6 +21,11 @@
 from .utils import Msg, get_hinted_fields
 
 
+def hide_last_line():
```

Comment:
This probably shouldn't be public

```suggestion
def _hide_last_line():
```